src/server/bin/plugin/pluginLogger.js

```diff
@@ -0,0 +1,41 @@
+﻿module.exports = function (silent) {
+  var previousLineEnded = true;
+  silent = !!silent;
+
+  function log(data, sameLine) {
+    if (silent) return;
+
+    if (!sameLine && !previousLineEnded) {
+      process.stdout.write('\n');
+    }
+
+    //if data is a stream, pipe it.
+    if (data.readable) {
+      data.pipe(process.stdout);
+      return;
+    }
+
+    if (!sameLine) data += '\n';
```

Comment:
Changed it to make an additional log call instead of appending to the payload variable.